### ISSUE TITLE
Looking for new maintainers

### ISSUE THREAD

**Issue by skywind3000** [skywind3000/atom-shell-commands#20]
For my personal sake, I am too busy to contribute in the future for a while,
and this repository is looking for new maintainers now.

Any one who would like to take over this plugin and make improvements for it 
can feed me back.

**Comment by lemongrassnginger** [comment]
@skywind3000 fair enough, we've all got busy lives. Until maintenance for this repo picks up again, for those looking for a package to run similar functionality, [process-palette](https://atom.io/packages/process-palette) looks like it's still being maintained regularly.

**Comment by dqsully** [comment]
I have a fork going, which I will release today (Edit: have released) under dqs-shell-packages. It should fix most of these issues, including allowing a `.shell-commands.cson` file to be placed in the project's root directory to specify commands, instead of just the user's config file.

**Comment by forhuan** [comment]
@skywind3000 抱歉 , 最近在准备考研，也没太有时间。
